pygmt/datasets/samples.py

```diff
@@ -142,39 +142,25 @@ def load_japan_quakes(**kwargs):
     return data
 
 
-def load_ocean_ridge_points(**kwargs):
+def _load_ocean_ridge_points():
     """
-    (Deprecated) Load a table of ocean ridge points for the entire world as a
+    Load a table of ocean ridge points for the entire world as a
     pandas.DataFrame.
 
-    .. warning:: Deprecated since v0.6.0. This function has been replaced with
-       ``load_sample_data(name="ocean_ridge_points")`` and will be removed in
-       v0.9.0.
-
     This is the ``@ridge.txt`` dataset used in the GMT tutorials.
 
```

Comment:
Please see the comments at https://github.com/GenericMappingTools/pygmt/pull/2301#discussion_r1062048779.
```suggestion
```